concent_api/common/tests/test_api_view.py

```diff
@@ -56,31 +58,9 @@ def setUp(self):
             "max_memory_size":      self.want_to_compute.max_memory_size,       # pylint: disable=no-member
             "num_cores":            self.want_to_compute.num_cores,             # pylint: disable=no-member
         }
-        deadline_offset = 10
-        message_timestamp = get_current_utc_timestamp() + deadline_offset
-        compute_task_def = tasks.ComputeTaskDefFactory(
-            deadline=message_timestamp,
-        )
-        task_to_compute = tasks.TaskToComputeFactory(
-            compute_task_def=compute_task_def,
-            requestor_public_key=encode_hex(REQUESTOR_PUBLIC_KEY),
-            provider_public_key=encode_hex(PROVIDER_PUBLIC_KEY),
-            price=0,
-        )
-        task_to_compute = load(
-            dump(
-                task_to_compute,
-                REQUESTOR_PRIVATE_KEY,
-                settings.CONCENT_PUBLIC_KEY,
-            ),
-            settings.CONCENT_PRIVATE_KEY,
-            REQUESTOR_PUBLIC_KEY,
-            check_time=False,
-        )
```

Comment:
It was not part of this task, but I found it, this block of code is not necessary. I deleted it, if I should'n please let me know